src/components/Keypad/Keypad.spec.js

```diff
@@ -22,15 +22,36 @@ describe('Keypad', () => {
         expect(subject.find('.numbers-container').length).toBe(1);
     });
 
-    it('renders the keypad values', () => {
-        expect(subject.find('.numbers-container').text()).toEqual('012');
+    it('renders the keys for each of the number values', () => {
```

Comment:
```suggestion
    it('renders a number of <Key/> components equal to total of numbers', () => {
```
Since this only tests the number of keys, maybe it's best to state it in this manner. I'm not in love with this description but I think you get what I mean 😅

Reply:
Actually, never mind. I can't think of a better way to write this since my above update doesn't mention that there are other `<Key/>`s for operators.